sources/platform/actors/publishing/monetize/pay_per_event.mdx

```diff
@@ -179,13 +179,28 @@ Your Actor might already have a start event defined, such as `actor-start` or an
 
 If you want to use the synthetic start event, remove the existing start event from your Actor and add the synthetic start event in Apify Console in the **Publication** tab.
 
+### Use synthetic default dataset item event `apify-actor-default-dataset-item`
+
+::::info Automatic per-item charging
+
+The `apify-actor-default-dataset-item` synthetic event charges users for each item written to the run's default dataset. It lets you align PPE pricing with per-result use cases without adding charging code to your Actor.
+
+::::
+
+#### How the synthetic default dataset item event works
+
+- Apify automatically charges this event whenever your Actor writes an item to the default dataset (for example, when using `Actor.pushData` without opening a named dataset).
```

Comment:
Suggestion: `without opening a named dataset` part is misleading. This only works for pushing into the **default** dataset.